code/modules/mob/living/recoil.dm

```diff
@@ -1,43 +1,89 @@
 /mob/living/proc/handle_recoil(var/obj/item/gun/G)
 	deltimer(recoil_reduction_timer)
-	if(G.one_hand_penalty)//If the gun has a two handed penalty and is not weilded.
+	if(G.one_hand_penalty) // If the gun has a two handed penalty and is not weilded.
```

Comment:
I know this is really superfluous but "weilded" is driving me crazy. Can you fix it to "wielded"?

Reply:
I did you extra too